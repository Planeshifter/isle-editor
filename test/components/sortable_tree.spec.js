// MODULES //

import React from 'react';
import renderer from 'react-test-renderer';
import SortableTree from '@isle-project/components/sortable-tree/main.js';


// TESTS //

describe( '<SortableTree />', function test() {
	it( 'renders a sortable tree', () => {
		const tree = renderer.create(
			<SortableTree
				onlyExpandSearchedNodes
				canDrag={false}
				treeData={[{
					title: 'Type of Data',
					children: [
						{
							title: 'Categorical',
							children: [{
								title: 'Chi-Square Test',
								subtitle: 'Test for independence for two variables or goodness-of-fit test'
							}]
						}
					]
				}]}
			/>
		).toJSON();
		expect( tree ).toMatchSnapshot();
	});
});
