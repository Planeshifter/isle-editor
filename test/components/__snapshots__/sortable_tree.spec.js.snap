// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SortableTree /> renders a sortable tree 1`] = `null`;
